src/it/MCOMPILER-474_recompile-dependent-when-package/dependent-module/src/main/java/Main.java

```diff
@@ -16,11 +16,10 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+package foo;
```

Comment:
Should we move file to `.../java/foo/Main.java` to be consistent with package name?

Reply:
We could, but there are tons of files to move manually for no benefit, and those are just tests, so I left it the way it is.  are you fancy doing it ?  I did not found  a trivial way to do that. 
Fwiw, the java compiler puts the compiled classes in the correct `foo/Main.class` while the groovy compile does not, so I had to move the groovy ones.